### PR TITLE
Update meraki ports

### PR DESCRIPTION
**Pull request by squintfox**
Updated missing and incorrect ports for Meraki switches.

### CODE REVIEW

File: device-types/Cisco/Meraki-MS225-48LP.yaml [hunk 1]
```diff
@@ -5,11 +5,16 @@ slug: ms225-48lp
 part_number: MS225-48LP-HW
 u_height: 1
 is_full_depth: false
+comments: '[MS225 Overview and Specifications](https://meraki.cisco.com/lib/pdf/meraki_datasheet_ms225.pdf)'
 power-ports:
   - name: Slot 1
     type: iec-60320-c14
     maximum_draw: 490
     allocated_draw: 53
+  - name: Cisco RPS 2300 connector for PSU failover
```
Comment (danner26) [inline device-types/Cisco/Meraki-MS225-48LP.yaml:14]: The name on this should be either what appears in  the config or printed on the device.. this name is a little too long to make sense

**Review by danner26 — CHANGES_REQUESTED** [review]

**Comment by danner26** [comment]
Otherwise everything looks good, thanks!

File: device-types/Cisco/Meraki-MS225-48LP.yaml (hunk 1 above)
Comment (squintfox) [reply device-types/Cisco/Meraki-MS225-48LP.yaml]: @danner26 I updated the name as to what's written in the documentation.  Unfortunately it's not a whole lot shorter.  It has no physical label on the device.

Comment (danner26) [reply device-types/Cisco/Meraki-MS225-48LP.yaml]: Why dont we call it something like `RPScon` then, its shorter and to the point that way

**Comment by danner26** [comment]
In regards to the `MS425-32`, I see that they are actually modular power supplies. These should be modelled as module-bays and then each specific PSU should be a module-type

File: device-types/Cisco/Meraki-MS225-48LP.yaml (hunk 1 above)
Comment (squintfox) [reply device-types/Cisco/Meraki-MS225-48LP.yaml]: Updated as "RPS Connector" which has some basis in technical documentation and is more brief that than the other options.

**Comment by squintfox** [comment]
@danner26 Updated as modular PSUs.

**Review by danner26 — APPROVED** [review]

**Pull request merged**